packages/server/src/controllers/files.ts

```diff
@@ -1,11 +1,45 @@
 import { Middleware } from 'koa'
+
 import { ObjectionModels } from '@aragonone/ipfs-background-service-shared'
 const { File } = ObjectionModels
+import { FilesValidator } from '../helpers'
+
+const dummy_file = {
+  sizeBytes: 5712538,
+  extension: "pdf",    
+  encoding: "UTF-8",
+  createdAt: 1590322097,
+  owner: "0x...",
+  verified: false,
+  transactionHash: "0x...",
+  cid: "Qm..."
+}
+
+export default class FilesController {
+  static create: Middleware = async (ctx) => {
+    FilesValidator.validateForCreate(ctx)
+    await File.create('test')
+    ctx.body = {
+      created: true
+    }
+  }
+
+  static unpin: Middleware = async (ctx) => {
```

Comment:
Are we planning to delete the file here? 
If not, are we going to have a separate delete endpoint? and probably it makes sense to allow pinning files back if we allow to unpin them :)

Reply:
hmm yes maybe let's just call this delete since I'm not handling pinning separately for now